### ISSUE TITLE
Provide language selection for TG

### ISSUE THREAD

**Issue by ronja-ui** [teragrep/ajs_01#244]
**Description**
To increase Teragrep's accessibility it should be translated to other languages (such as in Finnish) as well. 
 
**Use case or motivation behind the feature request**
There has been a request to provide Teragrep software in Finnish language. 

There are (probably) two ways to achieve this:

- Provide a select for language in profile settings
- Provide a separate installment for Finnish and English UI
 
**Related issues**

 
**Additional context**

**Comment by ronja-ui** [comment]
see related issue in zep_01 https://github.com/teragrep/zep_01/issues/18
